packages/text-bitmap/src/BitmapText.ts

```diff
@@ -474,7 +474,14 @@ export class BitmapText extends Container
         for (let i = 0; i < lenChars; i++)
         {
             const char = chars[i];
-            const xPos = (char.position.x + lineAlignOffsets[char.line]) * scale;
+            let offset = char.position.x + lineAlignOffsets[char.line];
+
+            if (this.roundPixels)
+            {
+                offset = Math.floor(offset);
+            }
+
+            const xPos = offset * scale;
```

Comment:
What if `scale` is a non-integer?

Reply:
When scaling / rotating a font you loose aliasing anyway. The roundPixels is good only if you work with 1.0 scale / 0.0 rotation.

Reply:
This sounds reasonable to me.